src/Internal/Types/TxConstraints.purs

```diff
@@ -417,6 +420,15 @@ mustMintCurrency
 mustMintCurrency mph =
   mustMintCurrencyWithRedeemer mph unitRedeemer
 
+mustMintCurrencyUsingNativeScript
+  :: forall (i :: Type) (o :: Type)
+   . NativeScriptHash
```

Comment:
Please use `NativeScript` itself instead of the hash, because it must be provided anyway. We are going to remove lookups in the future, so it's best to implement new interfaces without the need to use lookups.

Reply:
My bad I hadn't pushed my latest changes